src/components/pages/ParentHome/ParentCalendar.js

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+
+import 'antd/dist/antd.css';
+
+import { Calendar, Badge } from 'antd';
+
+function ParentCalendar() {
+  function getListData(value) {
+    let listData;
+    switch (value.date()) {
+      case 8:
+        listData = [
```

Comment:
This was an awesome choice, I feel as in the future we will be able to set `listData` to an array containing relevant course objects to have them display on certain days in the calendar.  Should tie in nicely. :)